pkg/receive/handler.go

```diff
@@ -54,10 +55,13 @@ const (
 	labelError   = "error"
 )
 
-// conflictErr is returned whenever an operation fails due to any conflict-type error.
-var conflictErr = errors.New("conflict")
+var (
+	// conflictErr is returned whenever an operation fails due to any conflict-type error.
+	conflictErr   = errors.New("conflict")
```

Comment:
I know it's not your issue but it's bit inconsistent naming (err in the end). Can we clean this up while we are here? (:

Reply:
I have already fixed this issue on another PR #2899